### PR TITLE
adjust size of stats_data/fsal_data

### PR DESCRIPTION
**Pull request by dvasilas**
This change increases the size of the fsal_data
array as we need to store additional data in it.

### CODE REVIEW

**Review by liaud — APPROVED** [review]

**Review by borisfaure — APPROVED** [review]

**Comment by oliviergaraud** [comment]
Now you need to update the commit in RING sub module and recompile/repackage ganesha.
(I can do this tomorrow)
If you decide to do it today you may want to also fix https://scality.atlassian.net/browse/RING-36625

File: src/include/fsal_api.h [hunk 1]
```diff
@@ -412,7 +412,7 @@ typedef enum {
 } stats_ops;
 
 
-#define FSAL_STAT_DATA_SIZE 64
+#define FSAL_STAT_DATA_SIZE 72
```
Comment (vietnscality) [inline src/include/fsal_api.h:415]: Why not a factor of block or power of 2?

Comment (dvasilas) [reply src/include/fsal_api.h:415]: I just set it to the exact size of the struct that should fit in. It shouldn't create alignment problems (I think).

Comment (vietnscality) [reply src/include/fsal_api.h:415]: Ah ok. That'd be useful to have a `static_assert` then ?

Comment (dvasilas) [reply src/include/fsal_api.h:415]: Yes there is already a `static_assert` on the ring side that checks that `sizeof` this array is `>=` `sizeof` the `struct` that must fit in it.

**Comment by dvasilas** [comment]
> Now you need to update the commit in RING sub module and recompile/repackage ganesha. (I can do this tomorrow) If you decide to do it today you may want to also fix https://scality.atlassian.net/browse/RING-36625

@oliviergaraud let's do the repackaging together tomorrow :upside_down_face: (I have some compilation errors due to missing deps)

File: src/include/fsal_api.h (hunk 1 above)
Comment (oliviergaraud) [reply src/include/fsal_api.h:415]: it's used and allocated within that struct: https://github.com/scality/nfs-ganesha/blob/V3-stable-scality/src/include/nfs_proto_data.h#L335

**Pull request merged**